src/main/scala/org/dia/loaders/RandomMatrixReader.scala

```diff
@@ -68,4 +71,19 @@ object RandomMatrixReader {
     (ndArray.data.asDouble, ndArray.shape)
   }
 
+
+  def createRandomSciDataset(name: String,
+                             variables : List[(String, Array[Int])],
+                             attributes : List[(String, String)]): SciDataset = {
+    val SciVars = variables.map({
+      case (filename, shape) =>
+        val nd4jTensor = new Nd4jTensor(Nd4j.rand(shape, filename.hashCode.toLong)) * 300.0
+        val dims = shape.zip(Array("u", "v", "w", "x", "y", "z"))
+          .map({ case (dim, name) => (filename + "_" + name.toString, dim) }).toList
+        (filename, new org.dia.core.Variable(filename, nd4jTensor, dims))
+    })
+
+    new org.dia.core.SciDataset(SciVars, attributes, name)
+  }
```

Comment:
Sorry there is no reason. I initially had this function written in the shell.
I've addressed it in the new PR.
Still working on finding where goddard moved their 3B42 files, so that still gives an error.